tests/env.run.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# This script can be used to warmup the environment and execute the tests
+# It is used by the docker image at startup
+
+if [[ ! -f .env ]]; then
+ cp .env.example .env
+fi
+
+#!/usr/bin/env bash
+START_TIME=$SECONDS
+
+if [ -z "${JAHIA_CONTEXT}" ];
```

Comment:
This is not needed anymore if you start using `JAHIA_URL` from the compose file. 

https://github.com/Jahia/sandbox/blob/main/tests/env.run.sh

Reply:
You would then construct `TEST_URL` (to be later used in the tests) based on `JAHIA_URL`